src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -279,6 +284,16 @@ class MessageHelper {
   static smart_objects::SmartObjectList CreateAddCommandRequestToHMI(
       ApplicationConstSharedPtr app, ApplicationManager& app_mngr);
 
+  /**
+   * @brief Creates Message for sending to HMI for unsubscribe from specified
+   * VehicleData. Puts in vehicle data types into message according to keys.
+   * @param message_to_hmi - message for sending.
+   * @param vehicle_data - collection with vehicle data types for ubsubscribe.
+   */
+  static void CreateUnsubscribeVehicleDataMessageForHMI(
```

Comment:
@IGapchuk make this function returning smart object as output value

Reply:
Done.

Reply:
Fixed in 73c67bf4f764f384b64cc45f9a7a3087eff31f1f.